source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -638,14 +850,34 @@ def _build_fallback(
                         fallback_dp_copy, fallback_type, requested_obj, via_poi_access, language
                     )
 
-                _update_fallback_sections(
-                    pt_journey,
-                    fallback_dp_copy,
-                    fallback_period_extremity,
-                    fallback_type,
-                    via_pt_access,
-                    via_poi_access,
-                )
+                if request["park_mode"] == [ParkMode.on_street.name] and (
+                    request["origin_mode"] == ["bike"] or request["destination_mode"] == ["bike"]
```

Comment:
```suggestion
                if ParkMode.on_street.name in request["park_mode"] and (
                    "bike" in request["origin_mode"] or "bike" in request["destination_mode"]
```

Reply:
request["park_mode"] can be null.

Reply:
the condition will be false if we do `first_section_mode[]=bike&first_seciont_mode[]=walking`, that's not what you want